buildpack.md

```diff
@@ -556,7 +556,10 @@ To choose an execution strategy,
 
 1. **IF** a buildpack-provided process type is chosen as the start command,
    1. **IF** the process type does not have `direct` set to `true`,
-      **THEN** the lifecycle MUST invoke the value of `command` as a command using Bash with values of `args` provided as arguments.
+      1. **IF** the process has one or more `args`
+         **THEN** the lifecycle MUST invoke a command using Bash, where `command` and each entry in `args` are shell-parsed tokens in the command.
+      2. **IF** the process has zero `args`
```

Comment:
What do you think about changing `1 + 2` to `if - else`?
I'm not sure what is more clear.

Reply:
I wanted to insert this change without changing the conventions in this part of the spec. The existing convention here is numbered **IF**s.